### ISSUE TITLE
OSS19 0x5b - adb

### ISSUE THREAD

**Issue by cpholguera** [OWASP/owasp-mstg#1228]
- Extend the adb section with a summary of features and basic usage.
- Search in the rest of the chapter for possible duplicates of adb descriptions, installation or usage. 
- Include the basics here and more "test case" specific content / commands in the test cases. For example: `adb backup` is very "test case" specific and it's correctly included in 0x05d Testing Backups for Sensitive Data. On the other side it would be a good idea to include `adb devices` here.

**Comment by cpholguera** [comment]
consider the additions from https://github.com/OWASP/owasp-mstg/pull/1280 once it is merged, so that we don't duplicate content: it talks mostly about `adb shell`.
